framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/settings/builder/KieServerHttpsS2ISettingsBuilderImpl.java

```diff
@@ -163,4 +164,17 @@ public KieServerS2ISettingsBuilder withKieServerSecret(String secret) {
         envVariables.put(OpenShiftTemplateConstants.KIE_SERVER_HTTPS_SECRET, secret);
         return this;
     }
+    
+    @Override
+    public KieServerS2ISettingsBuilder withTimerServiceDataStoreRefreshInterval(
+            Duration timerServiceDataStoreRefreshInterval) {
+        envVariables.put(OpenShiftTemplateConstants.TIMER_SERVICE_DATA_STORE_REFRESH_INTERVAL, Long.toString(timerServiceDataStoreRefreshInterval.toMillis()));
+        return this;
+    }
+    
+    @Override
+    public KieServerS2ISettingsBuilder withKieServerMemoryLimit(String limit) {
+        envVariables.put("KIE_SERVER_MEMORY_LIMIT", limit);
```

Comment:
please add key KIE_SERVER_MEMORY_LIMIT to the OpenShiftTemplateConstants class.

Reply:
Ok, thanks, done